lib/foreman/observable.rb

```diff
@@ -0,0 +1,24 @@
+module Foreman
+  module Observable
+    DEFAULT_NAMESPACE = :foreman
```

Comment:
Last minor thing can you make this "event.foreman" or "foreman.event"? Just in case we would use the API for something else in Foreman, this seems to be pretty generic.